russian/eating-walls/eating-marshmallows.md

```diff
@@ -4,4 +4,6 @@
 
 Ваши действия:
 
-[Позавтракать с Винипухом и Пяточком](breakfast/breakfast-marshmallows.md)
\ No newline at end of file
+[Позавтракать с Винипухом и Пяточком](breakfast/breakfast-marshmallows.md)
+[Вздремнуть после десерта](fall-asleep/fall-asleep.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.